llvm/lib/Target/SPIRV/SPIRVCallLowering.cpp

```diff
@@ -72,6 +72,47 @@ static ConstantInt *getConstInt(MDNode *MD, unsigned NumOp) {
   return nullptr;
 }
 
+// This code restores function args/retvalue types for composite cases
+// because the final types should still be aggregate whereas they're i32
+// during the translation to cope with aggregate flattening etc.
+static FunctionType *getFuncType(const Function &F) {
```

Comment:
I'd prefer `getFunctionType`, or even better `getOriginalFunctionType`

Reply:
Ok, I'll rename it.